README.md

```diff
@@ -36,8 +36,7 @@ changes may be written to a config file in a platform-specific location:
   $ go version
   $ go env GOROOT GOPATH
   ```
-  NOTE: `GOROOT` and `GOPATH` must not be on the same path. It is recommended
-  to add `$GOPATH/bin` to your `PATH` according to the Golang.org instructions.
+  NOTE: `GOROOT` and `GOPATH` must not be on the same path (if these are initialized).
```

Comment:
I would leave the recommendation there regardless.  It isn't only for linux, rather you need it in the path on other OSes too.

Reply:
It's just that on my Linux and Mac I observed both GOROOT and GOPATH to be empty, hence I thought it's kind of outdated recommendation,

anyway, I think it's worth mentioning these **might not be initialized** at all, so I rolled back and slightly rephrased it.